src/main/java/org/edumips64/Main.java

```diff
@@ -189,6 +189,10 @@ public static void main(String args[]) {
 
     showVersion();
 
+    // Set the Swing UI LookAndFeel according to the OS
+    try {
+        UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+    } catch (Exception e) {  }
```

Comment:
Please add a logging statement in the catch, instead of swallowing the exception.